torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -80,17 +123,10 @@ def forward(
         **kwargs,
     ) -> Union[Explanation, HeteroExplanation]:
 
-        if isinstance(index, Tensor) and index.numel() > 1:
-            if not self._support_multiple_indices():
-                raise ValueError(
-                    f"{self.attribution_method.__name__} does not support "
-                    "multiple indices. Please use a single index or a "
-                    "different attribution method.")
-
-        # TODO (matthias) Check if `internal_batch_size` can be passed.
-        if self.kwargs.get('internal_batch_size', 1) != 1:
-            warnings.warn("Overriding 'internal_batch_size' to 1")
-        self.kwargs['internal_batch_size'] = 1
+        if self._has_internal_batch_size():
```

Comment:
I think this check is not necessary. The method will anyways fail if we set `internal_batch_size` in `kwargs` if the captum attribution method doesn't have `internal_batch_size` as argument.